applications/StructuralMechanicsApplication/custom_utilities/shell_cross_section.cpp

```diff
@@ -49,79 +47,62 @@ void ShellCrossSection::BeginStack()
     if(!mEditingStack)
     {
         mEditingStack = true;
-        mThickness = 0.0;
         mStack.clear();
     }
 }
 
-void ShellCrossSection::AddPly(double thickness, double orientationAngle, int numPoints, const Properties::Pointer & pProperties)
+void ShellCrossSection::AddPly(const IndexType PlyIndex, int numPoints, const Properties::Pointer & pProperties)

```

Comment:
why did u remove the thickness? it was a fundamental feature i would say...

Reply:
It is NOT removed
It is now accesses through the Properties, the way it should be accesses